java/ql/src/Security/CWE/CWE-297/UnsafeHostnameVerification.ql

```diff
@@ -0,0 +1,75 @@
+/**
+ * @name Disabled hostname verification
+ * @description Accepting any certificate as valid for a host allows an attacker to perform a man-in-the-middle attack.
+ * @kind path-problem
+ * @problem.severity error
+ * @precision high
+ * @id java/everything-accepting-hostname-verifier
+ * @tags security
+ *       external/cwe/cwe-297
+ */
+
+import java
+import semmle.code.java.security.Encryption
+import semmle.code.java.dataflow.DataFlow
+import DataFlow::PathGraph
+import semmle.code.java.controlflow.Guards
```

Comment:
```suggestion

import semmle.code.java.controlflow.Guards
import semmle.code.java.dataflow.DataFlow
import semmle.code.java.security.Encryption

import DataFlow::PathGraph
```

Reply:
Applied in c69baaa1332665c55f7188057b39d0a4a066c88e